lib/puppet/parser/functions/zookeeper_servers_url.rb

```diff
@@ -0,0 +1,27 @@
+module Puppet::Parser::Functions
+  newfunction(:zookeeper_servers_url, :type => :rvalue, :doc => <<-EOS
+This function converts an array of zookeeper hostnames into a combined URL for
+zookeeper HA. You may pass a second argument to indicate the zookeeper port.
```

Comment:
I think thats from an older test I did and I forgot to remove that text, because the code does not parse the port, expects ipaddress:port (forces only 1 argument) and the documentation asks for and array of ipaddress:port.